src/components/shared/TractionFieldGroup.vue

```diff
@@ -54,6 +62,9 @@ export default {
         this.$emit('input', value)
       }
     },
+    getAttribute() {
```

Comment:
could this be a computed property rather than a method? Not sure just asking the question.

Reply:
Yes, you are right. Going through the documentation, it feels like getAttribute is better to be a computed property as it is just returning data from existing sources